tests/tools/updateManualTestcase.js

```diff
@@ -96,12 +84,13 @@ function toCBOR(root) {
     root = JSON.parse(JSON.stringify(root));
 
 // Now process the data and generate the correct cbor output
-    if ('entity' in root) {
-        tmp = fixFieldsForCBOR(root.entity);
+    if ('tx' in root) {
```

Comment:
I don't see why this change is necessary. I am probably missing something but it seems to be flipping the `if`.. 
if there is something else going on, it is probably relying on some kind of side effect, which is not so good I would say.

Reply:
It was linked to the structure I used in the template but it is outdated now.